addons/web_editor/controllers/main.py

```diff
@@ -688,20 +730,15 @@ def save_library_media(self, media):
 
         return attachments
 
-    @http.route("/web_editor/get_ice_servers", type='json', auth="user")
-    def get_ice_servers(self):
-        return request.env['mail.ice.server']._get_ice_servers()
+    @http.route("/web_editor/get_editor_collaboration_infos", type='json', auth="user")
```

Comment:
can we use a better name? (also, info/information is uncountable so never plural, just like data ^^)

maybe this?
```suggestion
    @http.route("/web_editor/edit/start", type='json', auth="user")
```